app/index.js

```diff
@@ -1,5 +1,9 @@
 const { app, ipcMain } = require('electron');
 const config = require('./config')(app.getPath('userData'));
+const Store = require('electron-store');
+const store = new Store({
+	name: 'tfl-settings'
```

Comment:
What tfl stands for? Can you maybe change this to be more descriptive?

Reply:
tfl => teams for linux. The electron-store uses config.json as default. yargs also the same. So avoid problems created a separate file for settings. Would you like to give it 'settings.json'?

Reply:
na, its ok. Probably update the readme in this folder/section to indicate we store the settings in there.